### DISCUSSION TITLE
[오픈소스] 한글 검색 라이브러리를 만들었습니다!

### DISCUSSION THREAD

**Discussion by Juhwa-Lee1023** [DeveloperAcademy-POSTECH/.github#77] [Show and tell]
안녕하세요. 1기 코인 입니다,

https://github.com/Juhwa-Lee1023/HangulSearch

이전에 만들었던 지하철 도착 정보를 위치기반으로 빠르게 확인할 수 있는 앱인 시티타이머에 지하철역명을 검색하는 기능을 구현하던 중, 팀원의 요청으로 초성 검색 기능을 추가했었습니다.

욕심이 생겨 한글의 특성을 살린 다양한 기능을 하나둘 추가하다 toss에서 만든 es-hangul을 보고 영감을 받아 라이브러리로 만들어 보았습니다.

HangulSearch는 초성 검색과 자동 완성 검색은 물론, 다양한 정렬 기능도 제공하고 있습니다. 이를 통해 iOS 환경에서 한글 검색을 보다 유연하고 직관적으로 사용할 수 있도록 설계했으니 많은 사용과 피드백 부탁드립니다!

또한, 데보션에도 이 라이브러리에 대한 소개 글을 작성해두었으니 관심 있으신 분들은 한 번씩 읽어봐 주시면 감사드리겠습니다.

https://devocean.sk.com/blog/techBoardDetail.do?ID=166056

감사합니다!

**Comment by yeniful** [comment]
안녕하세요 코인, 1기 예니입니다.
Swift Student Challenge 때 수상했던 앱을 시작으로 한글 앱, 라이브러리 전문가가 되어가시고 있군요!! 
한 분야에서 지속적으로 발전해나가시는 모습이 너무 멋져요 🤩
다음에 올려주신 라이브러리 사용해보고 후기 남겨보도록 하겠습니다 후후!!
